docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -54,6 +54,10 @@ The additional property sources are:
 An example would be properties from the Spring Cloud Config Server.
 See "`<<customizing-bootstrap-property-sources>>`" for how to customize the contents of this property source.
 
+NOTE:  Prior to Spring Cloud 2021.0.7 `PropertySourceLocators` (including the ones for Spring Cloud Config) were run during
```

Comment:
Not sure I like calling out specific version in the docs. Maybe, "previously"?

Reply:
But after the next release then that statement wont be correct

Reply:
@Buzzardo what do you suggest? Do you know what other teams do in these situations?